packages/bot-web-ui/src/components/load-modal/load-modal.scss

```diff
@@ -110,7 +110,7 @@
                 }
 
                 &-saved {
-                    margin-left: 1rem;
+                    margin-inline: 1rem;
```

Comment:
for margin-left it should be margin-inline-start or did you want to add margin for both left and right? @shafin-deriv

Reply:
I think margin-inline-start would be the right option here. Changing.